modules/system/assets/ui/storm-min.js

```diff
@@ -3296,7 +3296,7 @@ FilterWidget.prototype.getPopoverRangeTemplate=function(){return'
                                         type="text"                                                                       \
                                         name="date"                                                                       \
                                         value="{{ date }}"                                                                \
-                                        class="form-control align-right"                                                  \
+                                        class="form-control align-right popup-allow-focus"                                                  \
```

Comment:
@JLyons009 how do you get additional class `popup-allow-focus` added after re-compiling js and no adding it manually...I'm lost. Need to check!

Reply:
Not sure what caused that addition, but I resolved it!